src/workers/message-sender-234.js

```diff
@@ -9,8 +9,8 @@ const serviceMap = { nexmo, twilio }
 async function sendMessages() {
   const messages = await r.table('message')
     .getAll('QUEUED', { index: 'send_status' })
-    .filter((doc) => doc('user_number').match('[234]$'))
-    .group('user_number')
+    .filter((doc) => doc('contact_number').match('[234]$'))
```

Comment:
As far as I can tell, the message-sender workers are just broken up into arbitrary groups to spread the load among a few workers, and the last digit in a phone number has relatively even distribution.